web/Dockerfile

```diff
@@ -1,34 +1,31 @@
-FROM python:2.7
+FROM python:2.7-alpine3.6
 
 MAINTAINER Abhilash Raj
 
-ENV DEBIAN_FRONTEND=noninteractive
-# Add needed files for uwsgi server + settings for django 
-ADD mailman-web /opt/mailman-web
+# Add needed files for uwsgi server + settings for django
+COPY mailman-web /opt/mailman-web
 # Add startup script to container
-ADD assets/run.sh /opt/run.sh
+COPY docker-entrypoint.sh /usr/local/bin/
 
 # Install packages and dependencies for postorius and hyperkitty
 # Add user for executing apps, change ownership for uwsgi+django files and set execution rights for management script
-RUN apt-get update \
-    && apt-get install -y --no-install-recommends ruby-sass \
-                       wget \
-                       postgresql-client \
-    && rm -rf /var/lib/apt/lists/* \
-    && python -m pip install -U mailmanclient==3.1.0 \
-                             postorius==1.1.0 \
-                             hyperkitty==1.1.0 \
-                             django-mailman3==1.1.0 \
-                             whoosh \
-                             uwsgi \
-                             psycopg2 \
-                             dj-database-url \
-                             pymysql \
-	&& python -m pip install -U django==1.10 \
```

Comment:
If I'm correct, the upstream [does](https://gitlab.com/mailman/hyperkitty/blob/master/setup.py#L27) a job.